tasks/image-augmentation/main.py

```diff
@@ -75,10 +76,10 @@ def main() -> None:
         if noise is not None:
             secondAugmenters.append(iaa.AdditiveGaussianNoise(loc=0, scale=(0.0, noise*255), per_channel=0.5))
 
-        if blur is not None:
+        if blurPrc is not None:
             secondAugmenters.append(iaa.Sometimes(
-                blur,
-                iaa.GaussianBlur(sigma=(0, 0.5))
+                blurPrc,
+                iaa.GaussianBlur(sigma=(max(0, blurSigma - 5), blurSigma + 5))
```

Comment:
Let's be consistent with writing code. If you are writing code in a way that you are not using spaces around operators keep it that way and vice versa.